src/pe_source/dnstwistscript.py

```diff
@@ -0,0 +1,261 @@
+                        continue
+                    LOGGER.info("Running on %s", row["root_domain"])
+
+                    finalorglist = execute_dnstwist(root_domain)
+
+                    # Get subdomain uid
+                    sub_domain = root_domain
+                    try:
+                        sub_domain_uid = getSubdomain(PE_conn, sub_domain)[0]
+                    except Exception:
+                        # TODO: Create custom exceptions.
+                        # Issue 265: https://github.com/cisagov/pe-reports/issues/265
+                        LOGGER.info("Unable to get sub domain uid", "warning")
+                        # Add and then get it
+                        addSubdomain(PE_conn, sub_domain, pe_org_uid)
+                        sub_domain_uid = getSubdomain(PE_conn, sub_domain)[0]
+
+                    for dom in finalorglist:
+                        domain_dict, perm_list = checkBlocklist(
+                            dom, sub_domain_uid, source_uid, pe_org_uid, perm_list
+                        )
+                        if domain_dict is not None:
+                            domain_list.append(domain_dict)
+            except Exception:
+                # TODO: Create custom exceptions.
+                # Issue 265: https://github.com/cisagov/pe-reports/issues/265
+                LOGGER.info("Failed selecting DNSTwist data.", "Warning")
+                failures.append(org_name)
+                LOGGER.info(traceback.format_exc())
+
+            """Insert cleaned data into PE database."""
+            try:
+                cursor = PE_conn.cursor()
+                try:
+                    columns = domain_list[0].keys()
+                except Exception:
+                    LOGGER.critical("No data in the domain list.")
+                    failures.append(org_name)
+                    continue
+                table = "domain_permutations"
+                sql = """INSERT INTO {}({}) VALUES %s
+                ON CONFLICT (domain_permutation,organizations_uid)
+                DO UPDATE SET malicious = EXCLUDED.malicious,
+                    blocklist_attack_count = EXCLUDED.blocklist_attack_count,
+                    blocklist_report_count = EXCLUDED.blocklist_report_count,
+                    dshield_record_count = EXCLUDED.dshield_record_count,
+                    dshield_attack_count = EXCLUDED.dshield_attack_count,
+                    data_source_uid = EXCLUDED.data_source_uid,
+                    date_active = EXCLUDED.date_active;"""
+
+                values = [[value for value in dict.values()] for dict in domain_list]
+                extras.execute_values(
+                    cursor,
+                    sql.format(
+                        table,
+                        ",".join(columns),
+                    ),
+                    values,
+                )
+                PE_conn.commit()
+                LOGGER.info("Data inserted using execute_values() successfully..")
```

Comment:
Remove extra unnecessary period:
```suggestion
                LOGGER.info("Data inserted using execute_values() successfully.")
```